Description: disable the partition_rename_longfilename test as it fails
 when building with sbuild and schroots.
Author: Marc Deslauriers <marc.deslauriers@canonical.com>

Index: mysql-5.5-sid/mysql-test/t/disabled.def
===================================================================
--- mysql-5.5-sid.orig/mysql-test/t/disabled.def	2011-06-03 00:18:11.000000000 -0700
+++ mysql-5.5-sid/mysql-test/t/disabled.def	2011-06-03 00:52:58.756181167 -0700
@@ -17,3 +17,4 @@
 alter_table-big          : Bug#37248 2010-11-15 mattiasj was not tested
 create-big               : Bug#37248 2010-11-15 mattiasj was not tested
 archive-big              : Bug#11817185 2011-03-10 Anitha Disabled since this leads to timeout on Solaris Sparc
+partition_rename_longfilename : Fails when building with sbuild and schroots
